### PR TITLE
62 sign up and sign in form

### PR DESCRIPTION
**Pull request by iiinnnaaa**
Sign In and Sign Up forms config and styles

### CODE REVIEW

File: web/modules/custom/signin_popup/src/Plugin/Block/JoinBlock.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $is_anonymous = \Drupal::currentUser()->isAnonymous();
+
+    return [
+      '#theme' => 'join_now_block',
+      '#is_anonymous' => $is_anonymous,
```
Comment (leo72) [inline web/modules/custom/signin_popup/src/Plugin/Block/JoinBlock.php:25]: just write 
`'#is_anonymous'  => \Drupal::currentUser()->isAnonymous();`

**Review by leo72 — CHANGES_REQUESTED** [review]

File: web/themes/custom/drupalam/js/dialog.ajax.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+(function (Drupal) {
+  // Override core/dialog.ajax prepareDialogButtons behaviors
+  Drupal.behaviors.dialog.prepareDialogButtons = function prepareDialogButtons($dialog) {
+    // Do nothing = do not put buttons into dialog footer
```
[the remark this answers is not in the document]
Comment (iiinnnaaa) [reply web/themes/custom/drupalam/js/dialog.ajax.js:4]: Hi @leo72 , thanks for your review. With this code, I try to use the button that we already have in the form (and don't put it in  the ajax dialog footer), if it's the wrong way to do this I can try to change it for the login/registration form only

File: web/modules/custom/signin_popup/templates/join-now-block.html.twig [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<div class="join-block-wrapper">
+  {% if is_anonymous %}
+    <a href="{{ path('user.register') }}"
+       class="btn-join-now use-ajax text-decoration-none"
+       data-dialog-type="modal"
+       data-dialog-options="{&quot;dialogClass&quot;:&quot;signin-modal&quot;,&quot;width&quot;:1100,&quot;title&quot;: &quot;Welcome Back&quot;}">
```
[the remark this answers is not in the document]
Comment (iiinnnaaa) [reply web/modules/custom/signin_popup/templates/join-now-block.html.twig:6]: @leo72 can we discuss this part, please?

File: web/modules/custom/signin_popup/src/Plugin/Block/JoinBlock.php (hunk 1 above)
Comment (iiinnnaaa) [reply web/modules/custom/signin_popup/src/Plugin/Block/JoinBlock.php]: Okay, done

File: web/themes/custom/drupalam/drupalam.theme [hunk 4]
```diff
@@ -115,3 +124,26 @@ function drupalam_form_views_exposed_form_alter(&$form, &$form_state, $form_id)
...
+}
+
+/**
+ * Implements hook_preprocess_HOOK().
+ *
+ * Get current route name to check in login form template
+ *
+ */
+function drupalam_preprocess_form(&$variables){
+    $variables['route_name'] = \Drupal::routeMatch()->getRouteName();
```
[the remark this answers is not in the document]
Comment (iiinnnaaa) [reply web/themes/custom/drupalam/drupalam.theme:149]: Okay, I've fixed this too

**Pull request merged**